src/wp-includes/script-loader.php

```diff
@@ -2404,6 +2404,7 @@ function wp_global_styles_render_svg_filters() {
 	 */
 	if (
 		is_admin() &&
+		get_current_screen() != null &&
```

Comment:
```suggestion
		get_current_screen() !== null &&
```
While this is comparing to `null`, it will still generate a warning in PHPCS (not a failure though). Despite this, to keep noise down in logs, use the strictest comparison possible.